src/components/Pokedex/PokeList/PokeList.tsx

```diff
@@ -1,21 +1,13 @@
-import React from "react";
-import PokeCard from "../PokeCard/PokeCard";
-import "./PokeList.css";
+import React from 'react';
+import PokeCard from '../PokeCard/PokeCard';
+import './PokeList.css';
 
-
-export default function PokeList(props: any){
-    return (
-        <div className="pokelist">
-            {
-                props.list.map((pokeStuff: any) => {
-                    console.log(pokeStuff.id);
-                    return <PokeCard 
-                        id={pokeStuff.id}
-                        name={pokeStuff.name} 
-                        url={pokeStuff.sprites.front_default}
-                    />
-                })
-            }
-        </div>
-    )
-}
\ No newline at end of file
+export default function PokeList(props: any) {
+  return (
+    <div className="pokelist">
+      {props.list?.map((pokemon: any) => {
+        return <PokeCard pokemon={pokemon} />;
+      })}
```

Comment:
I changed this so that we're requesting information from within each `PokeCard` now. I like to keep the data as close to the component as possible, so in this case requesting each pokemon's data from within the card itself.

Reply:
I feel like this probably makes it easier to implement one of my suggestions from Discord, which is to not request all of these at once. If you just render 10 cards at a time, let's say, you automatically get request batching this way.